modules/cli/pages/cli-intro.adoc

```diff
@@ -63,3 +68,14 @@ If your Apple Silicon Mac does not already have Rosetta installed, it can be ins
 ----
 softwareupdate --install-rosetta
 ----
```

Comment:
Lines 64 through 70 are no longer needed since there is a macos arm64 package now.